hazelcast/src/main/java/com/hazelcast/map/impl/eviction/ExpirationManager.java

```diff
@@ -294,15 +295,23 @@ private boolean notHaveAnyExpirableRecord(PartitionContainer partitionContainer)
     }
 
     private Operation createExpirationOperation(int expirationPercentage, int partitionId) {
-        return new ClearExpiredOperation(expirationPercentage)
+        final ClearExpiredOperation clearExpiredOperation = new ClearExpiredOperation(expirationPercentage);
+        return clearExpiredOperation
```

Comment:
You can use `OperationService#invokeOnTarget` instead of `OperationService#execute` if you want automatic retry in case the exception is retryable, e.g. `PartitionMigratingException`. Or not if you just want the op to fail which is fine given that we retry later on.